### PR TITLE
Initial switch to pyup.io

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt r?

I've tested, with the exception of Python 3.5 and marionette support, which I haven't-yet been able to get to work:

https://pastebin.mozilla.org/8957102

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -3,17 +3,13 @@ envlist = py{27,35}, py{27,35}-marionette, flake8
 
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
-deps =
-    mock
-    pytest==3.0.4
-    pytest-selenium
-    pytest-xdist==1.15.0
+deps = -rrequirements/tests.txt
```
Comment (frankrousseau) [inline tox.ini]: It looks like there is a typo here. A space is missing between -r and requirements.

File: tox.ini [hunk 2]
```diff
@@ -3,17 +3,13 @@ envlist = py{27,35}, py{27,35}-marionette, flake8
...
 commands = pytest --ignore tests/marionette {posargs}
 
 [marionette]
 deps =
     {[testenv]deps}
-    marionette_driver==2.1.0
+    -rrequirements/marionette.txt
```
Comment (frankrousseau) [inline tox.ini]: It looks like there is a typo here. A space is missing between -r and requirements.

Comment (stephendonner) [inline tox.ini]: That's prevailing style: https://github.com/mozilla/PyPOM/commit/ad21988d0364843808c07f3aaa656c6c16b971b2

File: tox.ini (hunk 1 above)
Comment (stephendonner) [inline tox.ini]: Prevailing style: https://github.com/mozilla/PyPOM/commit/ad21988d0364843808c07f3aaa656c6c16b971b2

Comment (frankrousseau) [inline tox.ini]: Ok sorry, I didn't know that way of describing requirements.

Comment (stephendonner) [inline tox.ini]: @frankrousseau no worries; I appreciate you taking a look at the pull request - thanks!

File: tox.ini (hunk 2 above)
Comment (davehunt) [inline tox.ini]: Nice - I didn't know you could specify multiple requirements files. I was expecting that marionette.txt would simply import from tests.txt.

**Pull request merged**